akka-stream-tests/src/test/scala/akka/stream/impl/fusing/InterpreterSpec.scala

```diff
@@ -550,7 +550,9 @@ class InterpreterSpec extends AkkaSpec with GraphInterpreterSpecKit {
       new InvalidAbsorbTermination)) {
       lastEvents() should be(Set.empty)
 
-      EventFilter[UnsupportedOperationException]("It is not allowed to call absorbTermination() from onDownstreamFinish.", occurrences = 1).intercept {
+      // TODO we get two log entries after more verbose error logging was added to the interpreter
+      // couldn't figure out how to not do that without breaking other expected error logging
+      EventFilter[UnsupportedOperationException]("It is not allowed to call absorbTermination() from onDownstreamFinish.", occurrences = 2).intercept {
```

Comment:
The issue is error logging in `postStop()`. My original logging addition ensured that these are logged since they were impossible to propagate through the onError mechanism (we didn't log errors before, since the idea was that onError carries it anyway, but after completion there is no more onError). I am not sure this is actually needed anymore, since we always log now. You can probably remove that log statement and have your "catch-all" handle those cases instead.

What might be valuable though is to change the string to indicate if an error happened after the stage was logged and hence error was not propagated anywhere else.